api/handlers.go

```diff
@@ -371,25 +371,33 @@ func (api *API) userGET(u *database.User, w http.ResponseWriter, _ *http.Request
 func (api *API) userLimitsGET(_ *database.User, w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
 	// First check for an API key.
 	ak, err := apiKeyFromRequest(req)
+	respAnon := database.UserLimitsResponse{
```

Comment:
The types we respond with should be dedicated types in this file. It's not obvious for someone touching a type in `database` that the API response changes as well.

It's also not obvious to me how a number is going to be better than a string identifier for the tier from this MR.

Reply:
Watch this PR explode now!

I do agree, though. We should use dedicated types for data representation and dedicated types for user communication. I'll add those. It will blow up the PR but it's useful and it should also be easy to review.